docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/LoadingText.cs

```diff
@@ -55,11 +66,109 @@ public static void Example()
 
             // Load all files using path wildcard.
             var multipleFilesWildcardData = 
-                loader.Load(Path.Combine(dataDirectoryName, "*"));
+                loader.Load(Path.Combine(dataDirectoryName, "Data_*.csv"));
             PrintRowCount(multipleFilesWildcardData);
 
             // Expected Output:
             //   50
+
+
+            // Create a TextLoader with user defined type.
+            var loaderWithCustomType =
+                mlContext.Data.CreateTextLoader<Data>(hasHeader: false);
+
+            // Load a single file from path.
+            var singleFileCustomTypeData = loaderWithCustomType.Load(dataFiles[0]);
+            PrintRowCount(singleFileCustomTypeData);
+
+            // Expected Output:
+            //   10
+
+
+            // Create a TextLoader with unknown column length to illutrate
+            // how a data sample may be used to infer column size.
+            var dataSample = new MultiFileSource(dataFiles[0]);
+            var loaderWithUnknownLength = mlContext.Data.CreateTextLoader(
+                columns: new[]
+                {
+                    new TextLoader.Column("Features",
+                                            DataKind.Single,
+                                            new[] { new TextLoader.Range(0, null) })
+                },
+                dataSample: dataSample
+            );
+
+            var dataWithInferredLength = loaderWithUnknownLength.Load(dataFiles[0]);
+            PrintRowCount(dataWithInferredLength);
```

Comment:
>PrintRowCount [](start = 12, length = 13)

Should we print additional information here to "prove" that inferring the length works? For example, the type of the column of the schema of `dataWithInferredLength`.